Gopkg.toml

```diff
@@ -0,0 +1,34 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
+  version = "2.2.2"
+
+[prune]
+  go-tests = true
+  unused-packages = true
```

Comment:
you may also want:
```toml
  non-go = true
```

Reply:
Hmm, I didn't know about this (as this file was just auto-generated by dep --- dep being used only to pull in the  so that we can get golangci-lint to work out of the box). But at the same time I actually do not track what is in /vendor, so maybe it would be a NOP even with `non-go = true`? And like, if it ain't broke don't fix it... right?

Reply:
ok, FWIW the demo file includes this.

Reply:
Oh odd --- I guess dep changed behavior recently. Will add.